ci/test.sh

```diff
@@ -21,5 +41,11 @@ git config --global --add safe.directory "$(pwd)"
 # Enable IPv6
 echo 0 > /proc/sys/net/ipv6/conf/all/disable_ipv6
 
+env=""
+if [ ! -z $1 ]; then
+    check_valid_env $1
+    env="-$1"
+fi
+
 make storage
-make check
+make check${env}
```

Comment:
Run the test directly with tox instead:

    tox -e flake8,test-$env,bench-$env

Or change the makefile so we can pass the env:

    make check ENV=py38

If not specified, the makefile will run:

    tox

If specified, the makefile will run:

    tox -e flake8,test-$ENV,bench-$ENV

Reply:
Using the envs for the makefile feels like an abuse. I prefer to run `tox` directly.

Reply:
:+1: